backend/danswer/server/manage/search_settings.py

```diff
@@ -55,11 +55,10 @@ def set_new_embedding_model(
 
         embed_model_details.cloud_provider_id = cloud_id
 
+    # account for same model name being indexed with two different configurations
     if embed_model_details.model_name == current_model.model_name:
-        raise HTTPException(
-            status_code=status.HTTP_400_BAD_REQUEST,
-            detail="New embedding model is the same as the currently active one.",
-        )
+        if "__" not in current_model.model_name:
```

Comment:
would use the more specific pattern here (see comment above) and check for it explicitly.

Also what if they